### ISSUE TITLE
Virtual Machine screen sorting

### ISSUE THREAD

**Issue by heimdull** [kubevirt-manager/kubevirt-manager#92]
currently the Virtual Machine screen is sorted by Machines then VMs I would love to see a toggle for that and sort by just VMs like you do in VMware. If you have not used Vsphere there are 4 area or toggle that you can display VMs based on and thats Node, Vms, Storage and networks.

as a bonus it would also be nice to have a universal search box at the top where you can start typing the object then you get the list of VMs, networks, storage...

**Comment by feitnomore** [comment]
I'll figure something out here, I already have some ideas on my mind.
Thanks for suggesting this improvement.

**Comment by feitnomore** [comment]
Could get an initial prototype here... I'm working on it... This change will introduce some objects/buttons rearrangement, so I'm still figuring out how to arrange the new screen.

**Comment by feitnomore** [comment]
I've gone with option "1", and implemented a "refresh" button on the screens.
Would you be willing to give it a go? kubevirtmanager/kubevirt-manager:1.5.0-rc5

**Comment by youmailfreddy** [comment]
that looks much better and its super fast to get the list. Cluster I was testing with has 90 virtualmachines :) 

I do see something buggy with the icons for stop/start/console... where they are all right aligned and on top of each other so its a little buggy looking.
<img width="277" alt="image" src="https://github.com/user-attachments/assets/febc43ee-66b5-498b-a419-c147a6d3c86f" />

**Comment by heimdull** [comment]
that looks much better and its super fast to get the list. Cluster I was testing with has 90 virtualmachines :)

I do see something buggy with the icons for stop/start/console... where they are all right aligned and on top of each other so its a little buggy looking.
image

<img width="297" alt="image" src="https://github.com/user-attachments/assets/cbdb8c2e-7ea9-438e-b4e2-9a3af9ccf12a" />

**Comment by feitnomore** [comment]
Can you provide a full screenshot? 
I believe this is because of your resolution, could you try to change the zoom (ctrl -) to see how it behaves?

Thanks!

**Comment by heimdull** [comment]
yeah thats what it is, if I expand my browser it does show on one line for each host. two observations, do we need the status column since the first red/green icon is stopped/running? maybe just a hover tip on the icon? second was for the icons if they could be in a right click menu or dropdown or something ? 

I'm torn between a list ui like you have there and one where it looks more like vsphere with a vm drawer and then when you select a vm you get the controls on the right.

![image](https://github.com/user-attachments/assets/b1240c14-12ad-4c0a-9a13-b30258152208)

**Comment by feitnomore** [comment]
Great feedback... I don't plan to copy or do anything like vSphere, and I'm also not a good Front End guy, but I've got a few ideas that I'm implementing by this week. I'll probably release 1.5.0 by Sunday, with this new ideas.
Thanks!

**Comment by feitnomore** [comment]
I really appreciate you throwing ideas, and I am implementing most of them, however there are some that I am not skilled enougth to implement... I really wish I could get to a screen like the one you provided, but I'm really far from it...
Is there anything I can do to help your create vm workflow, to perform it on the User Interface?
Have you tried VM Pools? There is also Cluster API support, which I've been working and is really interesting.

Thanks!
